discovery-provider/scripts/test.sh

```diff
@@ -66,8 +67,10 @@ docker-compose \
 
 sleep 5
 
-# Unit tests
-pytest src
+if [ -z ${SKIP_TESTS+x} ]; then
```

Comment:
I don't understand this, is this set somewhere?

Reply:
The existing code is testing to see if the variable is set as opposed to it being non-empty:

https://stackoverflow.com/a/13864829

Thoughts?

Reply:
Very nice link, thanks, I think this is a nice way to do it.